### DISCUSSION TITLE
Error when installing arm64

### DISCUSSION THREAD

**Discussion by A5f3r** [xvzc/SpoofDPI#97] [Q&A]
When I type in the command: `curl -fsSL https://raw.githubusercontent.com/xvzc/SpoofDPI/main/install.sh | bash -s darwin-arm64`
Error: `tar: Error opening archive: Unrecognized archive format`

I have apple silicon, but when I use the Intel install command, I don't have this problem.

**Comment by AlexUnku** [comment]
Same issue here

**Comment by tollfree** [comment]
No issues for me now
